### ISSUE TITLE
Chrome becomes unresponsive with NVDA running

### ISSUE THREAD

**Issue by Knida89** [nvaccess/nvda#6791]
For the past several months Chrome would often become unresponsive unless I reset NVDA. It has now progressed where it still does this, but every once in awhile Chrome does not become responsive once NVDA is restarted. I have uninstalled and reinstalled Google Chrome, NVDA and Windows 10 but the issue persists. Any ideas?

**Comment by ehollig** [comment]
@Knida89, is this still an issue with the latest Chrome? It may help if you provided an NVDA log set to debug when performing the issue you describe.

**Comment by Knida89** [comment]
I did a clean install of Windows 10, and it still becomes unresponsive, but does become responsive again once I restart NVDA once or twice. I've attached a debug log. I also have a crash dump if that is needed.

**Comment by Knida89** [comment]
Here's the crash dump file.

**Comment by ehollig** [comment]
@Knida89 is this issue still happening? A lot of changes have occurred with NVDA and Chrome since 2017. Is this still happening with the latest versions of both programs?

**Comment by Adriani90** [comment]
@Knida89 are you still having this issue with NVDA 2019.1 nd Chrome 73? I cannot reproduce it. I can only imagine that it is due to the synthesizer you are using. Is the unresposiveness also occuring when you use eSpeak?

**Comment by Adriani90** [comment]
We do not have any updates from the initial author since more than 2 years. Closing as abandoned.
